server/app/routes/games.router.js

```diff
@@ -37,9 +36,19 @@ router.get('/:id', function(req, res, next){
 })
 
 router.post('/', function(req, res, next){
+  let invitedPlayers = req.body.players.invited.forEach(u=>u.id);
   Game.create(req.body.game)
-  .then(game=>game.setPlayers(req.body.players))
-  .then(game=> res.send(game))
+  .tap(game=>game.setInvitees(invitedPlayers))
+  .tap(game=>game.setPlayers(req.body.players.accepted))
+  .then(game=>game.setCommissioner(req.body.commissioner))
+  .tap(game=>{
+    let taskProms = req.body.tasks.map(taskObj=>Task.create(taskObj));
+    return Promise.all(taskProms).map(function(task, index) {
+        return task.setGame(game.id);
+    });
+  })
+  .tap(game=> res.send(game.id))
```

Comment:
I think if you `res.send` a number it'll attempt to send that as the status code *not* the body. Have you checked that this route handler is behaving as desired/expected?